### ISSUE TITLE
gtk2

### ISSUE THREAD

**Issue by stormchaser3000** [osendott/arch-frost-gtk-dark#3]
i know this is probably a known issue but the gtk2 part of this theme is the same as the light version of this theme. would be nice to have a gtk2 theme that fits this

<img src="http://u.cubeupload.com/stormchaser3000/201507130234531366x7.png"/>

**Comment by osendott** [comment]
Yeah, the theme started as a light-version only, and on request I threw together a dark version  very quickly. Shortly after, things got insane IRL and I just haven't had time to work on anything. Things are starting to calm down enough that I can get back to having a bit of free time and pick back up on themes and icons and such. If you happen to have already solved this problem, or do so before I have a chance, please feel free to submit a pull request and get your name in the credits :)

**Comment by stormchaser3000** [comment]
sadly i have no expirience with making gtk themes. :(

**Comment by stormchaser3000** [comment]
wait wth i didn't want to close this O_o

**Comment by stormchaser3000** [comment]
XD there
